README.md

```diff
@@ -45,10 +45,12 @@ Install freetype by brew.
 brew install freetype
 ```
 
+<!-- TODO: re-add this
```

Comment:
Note that this PR removes steps for deploying. 

Currently, creating release is not automated and is only sometimes released: https://github.com/openrr/urdf-viz/releases 
I'd like to automate that, but I'd like to fix the CI first because it can take time.

Reply:
I see.
It was tough to enable windows/mac deploying for me.
Good luck!